test/module/irohad/torii/command_sync_client_test.cpp

```diff
@@ -8,6 +8,7 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 #include "endpoint_mock.grpc.pb.h"
+#include "framework/mock_stream.h"
 #include "main/server_runner.hpp"
 #include "module/irohad/torii/torii_mocks.hpp"
```

Comment:
This include can be removed too, as well as `MockCommandServiceTransport` in that header.